api/server.js

```diff
@@ -2,6 +2,9 @@ const environment = require('./src/environment');
 const serverChecks = require('@medic/server-checks');
 const logger = require('./src/logger');
 const express = require('express');
+const morgan = require('morgan');
+const moment = require('moment');
+morgan.token('date', () => moment().format(logger.DATE_FORMAT));
```

Comment:
Can this be moved to `logger.js`? It feels like it fits more there.

Reply:
Done